README.md

```diff
@@ -40,7 +40,21 @@ For local development, we recommend to use the following setup which will enable
 pip install --upgrade pip
 git clone git@github.com:awslabs/syne-tune.git
 cd syne-tune
-pip install -e .[extra]
+pip install -e '.[extra]'
+```
+
+You may want to use environment management system such as Conda:
+```bash
+conda create -n syne-tune python=3.8
+conda activate syne-tune
+# run install comands here
+```
```

Comment:
Adding instructions for conda is a bit arbitrary, can we ommit it? (this raises the question why using conda and not venv, also we have to maintain those instructions)

Reply:
I am (very much) against conda, having had trouble with it in the past.

Reply:
Alright, will remove.